config/application.yml.example

```diff
@@ -9,6 +9,7 @@
 # Figaro requires all values to be explicit strings.
 
 email_from: 'no-reply@login.gov'
+exception_recipients: '["user1@example.com", "user2@example.com"]'
```

Comment:
Should we make this a comma separated string and then use `split(',')` instead of `JSON.parse` in the initializer?

Reply:
I'm inclined to say "yes".